Dockerfile

```diff
@@ -35,20 +26,24 @@ FROM debian:bookworm-slim
 ENV DEBIAN_FRONTEND=noninteractive
 ENV TZ=Etc/UTC
 
-# trunk-ignore(terrascan/AC_DOCKER_0002): Known terrascan issue
-# trunk-ignore(hadolint/DL3008): Use latest version of packages for buildchain
-RUN apt-get update && apt-get --no-install-recommends -y install libc-bin libc6 libgpiod2 libyaml-cpp0.7 libulfius2.7 libusb-1.0-0-dev liborcania2.3 libssl3 && \
-    apt-get clean && rm -rf /var/lib/apt/lists/*
+USER root
```

Comment:
@fifieldt I've re-added the `USER` directive down here. Semgrep should only care about the resultant (bottom-most) image.

Reply:
All trunk / semgrep warnings should be handled now :)